### DISCUSSION TITLE
How can access the API's of pocketBase

### DISCUSSION THREAD

**Discussion by PratyushSawan** [pocketbase/pocketbase#1563] [Q&A]
I am running pocketbase on my local windows machine.

![image](https://user-images.githubusercontent.com/40737575/211701585-738177ee-c857-4621-9612-84f474074338.png)

I created some of the users in the Admin UI dashboard

![image](https://user-images.githubusercontent.com/40737575/211701763-0f75bfa5-3295-431d-815d-89e551c666b1.png)

Now I want to know,can I directly able to access the API's endpoint to get the users list as an admin from postman directly or I have to make a specific API using node to access the list of user?

Because as of now I am able to see that in the console there is a endpoint is showing for call the API. please guide me how can I access the API's
![image](https://user-images.githubusercontent.com/40737575/211702092-09d244e5-9324-45c3-a569-6df13dc3d3a0.png)

**Comment by avnav0** [comment]
hey man didnt notice this thread, and i put one up after you - have been having the same issues.  it's very frustrating if you are not a javascript guy.

`	data = {"identity": 'test@gmail.com',
	        "password": "test"}
	response = requests.post(url='http://127.0.0.1:8090/api/admins/auth-with-password',
	                  data=data)
	return response`

i cannot find any api documentaiton - which is kind of the whole point of this db..

**Comment by ganigeorgiev** [comment]
@avnav88, @PratyushSawan

You can find API documentation for each of your collections by clicking on the "API Preview" button in the Admin UI OR exploring the "Web API" Docs in https://pocketbase.io/docs/api-records/.

**Reply by PratyushSawan** [reply]
I went through the process. But where I have to use JS syntax which is provided over the API doc. I am running the server in windows machine and it gives a base URL for hitting the API. So, can I use that same Base URL for the API calling by using the given endpoint which I get from the "API Preview".

I am confused what I have to do next after start the pocketBase server for checking the API in postman. Should I have to write some other API's in node.js using that JS syntax which is provide over the "API Preview" doc.

**Reply by ganigeorgiev** [reply]
I don't understand what do you mean.

Both the "API preview" and the general docs in https://pocketbase.io/docs/api-records/ contains:
- SDKs examples
- Information about the underlaying http api (request url, required body parameters, etc.)

![docs](https://i.imgur.com/oVPugOF.png)

**Reply by avnav0** [reply]
yep figured it out after tinkering around with the ui - didn't see any note about the preview feature in the docs so it took a while.

really appreciate this tech, but i think better api docs could help people save a lot of time.  at least for less experienced folks like myself.

for example, using python, i have to translate this to the requests library.  and if i try to login as admin and get a token - i have to pass the credentials through the data parameter.  if i want to insert data into a collection, i do that through the json attribute.  but i had to figure those out the hard way. :/

maybe there's a way to show example post requests similar to how the reponses are shown in the preview?
